packages/alfa-aria/src/attributes.ts

```diff
@@ -90,6 +90,14 @@ export const DescribedBy: Attribute = {
   type: "id-reference-list"
 };
 
+/**
+ * @see https://www.w3.org/TR/wai-aria/#aria-describedat
+ */
+export const DescribedAt: Attribute = {
+  name: "aria-describedat",
+  type: "uri"
+};
+
```

Comment:
This will need to be marked as obsolete and is not to be considered a valid ARIA attribute.